include/interaction/LabelsManager.h

```diff
@@ -0,0 +1,93 @@
+﻿//  ----------------------------------
+//  Copyright © 2015, Brown University, Providence, RI.
+//  
+//  All Rights Reserved
+//   
+//  Use of the software is provided under the terms of the GNU General Public License version 3 
+//  as published by the Free Software Foundation at http://www.gnu.org/licenses/gpl-3.0.html, provided 
+//  that this copyright notice appear in all copies and that the name of Brown University not be used in 
+//  advertising or publicity pertaining to the use or distribution of the software without specific written 
+//  prior permission from Brown University.
+//  
+//  See license.txt for further information.
+//  
+//  BROWN UNIVERSITY DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE WHICH IS 
+//  PROVIDED “AS IS”, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+//  FOR ANY PARTICULAR PURPOSE.  IN NO EVENT SHALL BROWN UNIVERSITY BE LIABLE FOR ANY 
+//  SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR FOR ANY DAMAGES WHATSOEVER RESULTING 
+//  FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR 
+//  OTHER TORTIOUS ACTION, OR ANY OTHER LEGAL THEORY, ARISING OUT OF OR IN CONNECTION 
+//  WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. 
+//  ----------------------------------
+//  
+///\file Labels.h
+///\author Benjamin Knorlein
+///\date 6/25/2019
+
+#ifndef LABELSMANAGER_H
+#define LABELSMANAGER_H
+#include <vector>
+#include <glm/mat4x4.hpp>
+#include <string>
+#include <map>
+
+
+class Model;
+class Texture;
+class VRVolumeApp;
+class ShaderProgram;
+
+struct LabelBillboard
+{
+  unsigned int line_vba;
+  Texture* label_texture;
+  Model* label_model;
+  glm::vec3 position;
+  unsigned int volume_id;
+};
+
+class LabelsManager
+{
+public:
+  LabelsManager(VRVolumeApp& vrapp,ShaderProgram& lines_shader, ShaderProgram& plane_shader);
+  ~LabelsManager();
+
+  
+  void add(std::string texture, float x, float y, float z, float textPosZ, float size, float off_set, int volume);
```

Comment:
Is there a reason there's two lines of space here? I would think it should only be one, not sure if the formatter will do that for you